src/app/components/client/ExposureCard.tsx

```diff
@@ -278,3 +361,13 @@ export const ExposureCard = (props: ExposureCardProps) => {
 
   return exposureCard;
 };
+
+function formatOtherBreachCategoriesLabel(sentence: string) {
+  const words = sentence.split("-");
+  const formattedWords = words.map((word) => {
+    const firstLetter = word.charAt(0).toUpperCase();
+    const restOfWord = word.slice(1);
+    return firstLetter + restOfWord;
+  });
+  return formattedWords.join(" ");
+}
```

Comment:
nit: not sure how well this will localize (unless we only care about en-us, but still worth pointing out maybe)